pd/src/state/jellyfish.rs

```diff
@@ -17,39 +14,26 @@ pub enum Key {
 }
 
 impl Key {
-    pub fn hash(self) -> HashValue {
+    pub fn hash(self) -> KeyHash {
         match self {
             Key::NoteCommitmentAnchor => {
-                let mut state = NoteCommitmentAnchorHasher::default();
-                state.update(b"");
-                state.finish()
+                let mut state = sha2::Sha256::new();
+                state.update(b"nct");
+                KeyHash(*state.finalize().as_ref())
             }
         }
     }
 }
```

Comment:
I think we can get rid of the `Key` type entirely here and just write the NCT data in under the key `b"nct"`?